internal/resolver/resolver.go

```diff
@@ -148,8 +148,8 @@ func (r *Resolver) Resolve(reqs []*chart.Dependency, repoNames map[string]string
 //
 // This should be used only to compare against another hash generated by this
 // function.
-func HashReq(req []*chart.Dependency) (string, error) {
-	data, err := json.Marshal(req)
+func HashReq(req []*chart.Dependency, lock []*chart.Dependency) (string, error) {
```

Comment:
nit: could be replaced with `func(req, lock []*chart.Dependency)`

Reply:
Fixed it!